wallet-new/server/Cardano/Wallet/LegacyServer.hs

```diff
@@ -23,3 +24,4 @@ walletServer :: ( Migration.HasConfigurations
 walletServer natV0 diffusion =
          V0.handlers natV0 diffusion
     :<|> V1.handlers natV0 diffusion
+    :<|> Dev.handlers natV0
```

Comment:
We should add a combinator like `developmentOnly` which would wrap handlers so that we cannot call them if we are not running in debug mode? Something like:

```
:<|> developmentOnly (Development.handlers natV0)
```

Reply:
Good idea!